creator-node/package.json

```diff
@@ -12,8 +12,11 @@
     "test:unit": "./scripts/run-tests.sh unit_test",
     "coverage": "nyc npm run test",
     "report": "nyc report --reporter=html",
-    "lint": "./node_modules/.bin/standard",
-    "lint-fix": "./node_modules/.bin/standard --fix"
+    "lint": "npm run prettier:check",
+    "lint:fix": "npm run prettier:write",
+    "prettier:base": "./node_modules/.bin/prettier --parser typescript --single-quote --no-semi --trailing-comma none",
```

Comment:
am a `prettier` noob but could these flags not be specified in `prettierrc` instead? confusing to have configs specified in multiple places

Reply:
i'll experiment more with prettier rules and see if i can consolidate